app/views/users/register.haml

```diff
@@ -20,6 +20,11 @@
       = f.label :first_time_attendee, 'Will this be your first time at beCamp?'
       = f.select :first_time_attendee, options_for_select([['Please choose one...', ''], ['Yes', true], ['No', false]], current_user.first_time_attendee)
 
+    = field_set_tag do
+      = f.label :list_me, 'Can we show your name?'
+      = f.radio_button :list_me, "True", :checked => true
```

Comment:
I really like seeing who is coming to beCamp, I know my excitement level is based on seeing who else is showing up, and I believe that attracts other folks, hence the checked => true.